android/modules/ui/src/java/ti/modules/titanium/ui/widget/listview/ListViewProxy.java

```diff
@@ -314,6 +314,12 @@ public void deleteSectionAt(int index, @Kroll.argument(optional = true) KrollDic
 		}
 	}
 
+	// not needed on Android but parity for iOS method call
+	@Kroll.method
+	public void deselectItem(int section, int item)
```

Comment:
I would expect an empty-function warning or something. Isn't that the case?

Reply:
I thought it might be a good empty function as you have to use it on iOS so often and you always have to make the `if OS_IOS` check before it because it will throw a "method not found" error on Android.

but as you've said: no requests by other people yet, just thought it might be a good convenient method

Reply:
You can just write `$.listView.deselectItem?.(event.sectionIndex, event.itemIndex)` for that :)